src/parameters/parameters_physics_model.cpp

```diff
@@ -45,6 +53,28 @@ void PhysicsModelParam::declare_parameters (dealii::ParameterHandler &prm)
                               dealii::Patterns::Double(1e-15, dealii::Patterns::Double::max_double_value),
                               "Ratio of the large eddy simulation filter width to the cell size (default is 1)");
 
+            prm.declare_entry("do_compute_filtered_solution", "false",
+                              dealii::Patterns::Bool(),
+                              "Flag to compute the filtered solution. By default, false.");
+
+            prm.declare_entry("apply_modal_high_pass_filter_on_filtered_solution", "false",
+                              dealii::Patterns::Bool(),
+                              "Flag to apply a modal high pass filter the filtered solution. By default, false.");
+
+            prm.declare_entry("poly_degree_max_large_scales", "0",
+                              dealii::Patterns::Integer(0, dealii::Patterns::Integer::max_int_value),
+                              "Used for variational-multiscale (VMS) filtering of the solution. "
+                              "This represents the maximum polynomial order for the large scales. "
+                              "Warning: This must be less than the poly_degree of the solution.");
+
+            prm.declare_entry("dynamic_smagorinsky_model_constant_clipping_limit", "0.01",
+                              dealii::Patterns::Double(1e-15, dealii::Patterns::Double::max_double_value),
+                              "Clipping limit for the Dynamic Smagorinsky model constant (default is 0.01).");
+
+            prm.declare_entry("apply_low_reynolds_number_eddy_viscosity_correction", "false",
+                              dealii::Patterns::Bool(),
+                              "Flag for applying the low Reynolds number eddy viscosity correction. By default, false.");
+
```

Comment:
add something in modify parameters for setting this to true for a given Reynolds number